docs/PORTABLE_SERVICES.md

```diff
@@ -6,10 +6,10 @@ layout: default
 
 # Portable Services Introduction
 
-This systemd version includes a preview of the "portable service"
-concept. "Portable Services" are supposed to be an incremental improvement over
-traditional system services, making two specific facets of container management
-available to system services more readily. Specifically:
+systemd since version v239 includes a new "portable service" concept.
+"Portable Services" are an incremental improvement over traditional system
+services, making two specific facets of container management available to
+system services more readily. Specifically:
```

Comment:
This is awkward too. Maybe:
`"Portable Services" are a delivery method for system services that uses two specific features of container management:`
`1. Applications are bundled. I.e. multiple services, their binaries… are packaged…`